tools/lib/provision.py

```diff
@@ -331,6 +333,7 @@ def main(options):
         else:
             print("RabbitMQ is already configured.")
 
+        run(["sudo", "service", "postgresql", "start"])
```

Comment:
Hmm. How about we put this up in the block with the other `service` commands (`memcached`, etc.)? That'd be cleaner, I think.

And I guess we might as well switch it to `restart` to match the others.